charts/nitro/templates/validator/deployments.yaml

```
@@ -0,0 +1,166 @@
+{{- if .Values.validator.enabled }}
+{{- range .Values.validator.splitvalidator.deployments }}
+{{- $global := deepCopy $.Values.validator.splitvalidator.global }}
+{{- $deployment := . }}
+{{- $mergedValues := merge $global $deployment }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "nitro.fullname" $ }}-val-{{ .name }}
+  labels:
+    {{- include "nitro.labels" $ | nindent 4 }}
+    {{- with $mergedValues.extraLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  replicas: {{ .replicas | default $mergedValues.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "nitro.selectorLabels" $ | nindent 6 }}
+      validator: {{ .name | quote }}
+  template:
+    metadata:
+      annotations:
+        {{- with $mergedValues.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+        checksum/configmap: {{ include (print $.Template.BasePath "/validator/configmap.yaml") $ | sha256sum }}
+      labels:
+        {{- include "nitro.selectorLabels" $ | nindent 8 }}
+        validator: {{ .name | quote }}
+        function: split-validator
+    spec:
+      serviceAccountName: {{ include "nitro.serviceAccountName" $ }}
+      nodeSelector:
+        {{- with $mergedValues.nodeSelector }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      terminationGracePeriodSeconds: {{ $mergedValues.terminationGracePeriodSeconds | default 60 }}
+      affinity:
+        {{- with $mergedValues.affinity }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      tolerations:
+        {{- with $mergedValues.tolerations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      topologySpreadConstraints:
+        {{- if $mergedValues.topologySpreadConstraints }}
+        {{- range $constraint := $mergedValues.topologySpreadConstraints }}
+        - {{ toYaml $constraint | nindent 10 | trim }}
+          {{- if not $constraint.labelSelector }}
+          labelSelector:
+            matchLabels:
+              {{- include "nitro.selectorLabels" $ | nindent 14 }}
+              validator: {{ .name | quote }}
+              function: arb-validator
```

should this not be `split-validator` so it matches the labels above?
```suggestion
              function: split-validator
```